.circleci/config.yml

```diff
@@ -129,12 +138,13 @@ jobs:
           $ProgressPreference = "SilentlyContinue"
           Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
           choco install cmake --installargs '"ADD_CMAKE_TO_PATH=System"'
-          choco install boost-msvc-14.1
+          # choco install boost-msvc-14.1
           choco install doxygen.install
           choco install nsis.portable
           refreshenv
           $qt_url = 'https://github.com/martinrotter/qt5-minimalistic-builds/releases/download/5.12.9/qt-5.12.9-dynamic-msvc2019-x86_64.7z'
           $sdl2_url = 'https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip'
+          $boost_url = 'https://github.com/ahmedskhalil/Boost-1.73-Prebuilts/releases/download/0.0.1/boost_1_73_0.7z'
```

Comment:
What was our conclusion on this from our last conversation?  Was it to just include the zip files in the SmartPeak repo?

Reply:
Github excludes file sizes that exceeds the 100MB mark [Conditions for large files](https://docs.github.com/en/free-pro-team@latest/github/managing-large-files/conditions-for-large-files). OpenMS prebuilt contribs do include the required boost libraries except for `boost::filesystem` so once we get away from it and move to the `std::filesystem` we can rely solely on OpenMS prebuilt contribs for both OpenMS and SmartPeak.

Reply:
Thanks for that clarification.  OK let's go with this as a temporary solution for now until C++17 is integrated.